cql3/statements/select_statement.cc

```diff
@@ -56,6 +56,7 @@
 #include "partition_slice_builder.hh"
 #include "cql3/untyped_result_set.hh"
 #include "db/timeout_clock.hh"
+#include "db/system_keyspace.hh"  // Decl. of `is_system_keyspace()'
```

Comment:
Please remove this comment.

Reply:
Better yet, forward declare this function only. Ref: 4f6b892aa1fb994c7ad6f19b8e5c3b660ff8f5c6